test/hooks.js

```diff
@@ -15,18 +15,17 @@
 /* global beforeEach afterEach */
 'use strict';
 
-const { CloudantV1 } = require('@ibm-cloud/cloudant');
-const url = new URL((process.env.COUCH_BACKEND_URL) ? process.env.COUCH_BACKEND_URL : 'https://no-couch-backend-url-set.test');
-const { BasicAuthenticator, NoAuthAuthenticator } = require('ibm-cloud-sdk-core');
-const authenticator = (url.username) ? new BasicAuthenticator({ username: url.username, password: decodeURIComponent(url.password) }) : new NoAuthAuthenticator();
-const serviceOpts = {
-  authenticator
-};
-const cloudant = new CloudantV1(serviceOpts);
-// Remove auth from URL before using for service
-cloudant.setServiceUrl(new URL(url.pathname, url.origin).toString());
 const uuid = require('uuid').v4;
 const fs = require('fs');
+const { newSimpleClient } = require('../includes/request.js');
+
+function sharedClient() {
```

Comment:
This is in conjunction with the other suggestions to export/import the const. Otherwise we'll initialize a new client every time we use this, which is probably even less sharing than before.

```suggestion
const sharedClient = makeSharedClient();
function makeSharedClient() {
```